annotation/tests/test_jobs.py

```diff
@@ -182,12 +186,11 @@ def test_check_annotators(
         check_annotators(annotators, validation_type)
 
 
-def test_collect_job_names_all_db():
+def test_collect_job_names_all_db(jobs_to_test_progress: Tuple[Job]):
```

Comment:
```suggestion
def test_collect_job_names_all_db(jobs_to_test_progress: Tuple[Job, ...]):
```
[link](https://docs.python.org/3/library/typing.html#:~:text=To%20denote%20a%20tuple%20which%20could%20be%20of%20any%20length%2C%20and%20in%20which%20all%20elements%20are%20of%20the%20same%20type%20T%2C%20use%20tuple%5BT%2C%20...%5D.%20To%20denote%20an%20empty%20tuple%2C%20use%20tuple%5B()%5D.%20Using%20plain%20tuple%20as%20an%20annotation%20is%20equivalent%20to%20using%20tuple%5BAny%2C%20...%5D%3A)